plugins/node/opentelemetry-instrumentation-mongodb/src/mongodb.ts

```diff
@@ -142,14 +142,7 @@ export class MongoDBInstrumentation extends InstrumentationBase<
           typeof resultHandler !== 'function' ||
           typeof ops !== 'object'
         ) {
-          return original.call(
-            this,
-            server,
-            ns,
-            ops,
-            typeof options === 'function' ? callback : options,
-            callback
-          );
+          return original.call(this, server, ns, ops, options, callback);
```

Comment:
shouldn't we the same if as in the other cases also here?

Reply:
Done